src/Toast/ToastContent.tsx

```diff
@@ -78,14 +109,17 @@ ToastContent.displayName = 'ToastContent';
 /**
  * STYLE
  */
-export const toastContentStyles = (theme: ITheme['toast']): IBoxCSS => {
+export const toastContentStyles = (props: IToastContentProps, theme: ITheme['toast']): IBoxCSS => {
+  const type: any = props.type;
```

Comment:
```suggestion
  const type: keyof ITheme['toast'] = props.type;
```
Don't remember whether toast exists on the object, if not, rename it.

Reply:
added